### PR TITLE
Maximum distance between same elements in Python

### LINKED ISSUE #6296: Maximum distance between same elements  in Python
## 🚀 Feature

(A clear and concise description of what the feature is.)

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

yes

## Motivation

(Please outline the motivation for the proposal.)

## Pitch

(Please explain why this feature should be implemented and how it would be used.)

### PR DESCRIPTION
**Pull request by arpit456jain**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description

Let say an array is given with repeated elements, we have  to find the maximum distance between two occurrences of an element.

eg:
[1, 1, 2, 2, 2, 1]

ans : 5 (between two ones first and last)
Distance for 1 is: 5-0 = 5
### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
Fixes : #6296 
(Write your answer here.)

### CODE REVIEW

**Comment by arpit456jain** [comment]
@ankitaggarwal23 please review

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Readme file is buggy fix it

**Comment by arpit456jain** [comment]
@ankitaggarwal23 done

File: Python/README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@
 * [Right view of a binary tree](ds/Right_view_of_binary_tree.py)
 * [Rotate the array by k elements](ds/rotate_the_array_by_k_elements.py)
 * [Stack using list](ds/stack_using_list.py)
+* [Max distance between same elements](ds/MaxDistanceBetweenSameElements.py)
```
Comment (ankitaggarwal23) [inline Python/README.md]: Write it as `Maximum`

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**